src/core/wrap.ts

```diff
@@ -11,5 +13,5 @@ export type IfSingle<T extends string, Yes, No> = StripEscapes<T> extends `${inf
 
 export const wrap = (s: string | Input<any>) => {
   const v = s.toString()
-  return v.replace(/^\\/, '').length === 1 ? v : `(${v})`
+  return /^((?<!\\)(\().*(?<!\\)(\))|.|\\[^()])$/.test(v) ? v : `(${v})`
```

Comment:
What about:

```suggestion
  return /^(\(.*\)|\\?.)$/.test(v) ? v : `(${v})`
```

We can also slightly improve perf by removing it out of the function.

Reply:
I forget to check taht we can totally remove the lookbehinds 🤦‍♂️ 
thanks for pointing that out!